frontend/public/components/utils/log-window.jsx

```diff
@@ -50,8 +51,13 @@ export class LogWindow extends React.PureComponent {
   }
 
   _handleScroll() {
-    // Stream is finished, take no action on scroll
-    if (this.props.status === STREAM_EOF) {
+    const horizontalScroll = this.state.prevScrollLeft !== this.scrollPane.scrollLeft;
+
+    // Stream is finished or horizontal scroll, take no action
+    if (this.props.status === STREAM_EOF || horizontalScroll) {
+      if (horizontalScroll) {
```

Comment:
Is this wrong if horizontalScroll is 0?

Reply:
`horizontalScroll` is a boolean, set on first line of the function.

Reply:
Sorry, I see now. We might make it something like `scrollLeftChanged` for people like me who have trouble reading :)

I don't really see a reason for this block to be inside the `if`... I would pull it outside to avoid the nesting, which makes it a little harder to understand.

Reply:
ok, seems redundant though.  Want to `return` for both conditions.

Reply:
fixed